### PR TITLE
Updates Glossary with PEF.

### PR DESCRIPTION
**Pull request by andrebonon**

### CODE REVIEW

File: content/en/docs/glossary/_index.md [hunk 1]
```diff
@@ -75,6 +75,9 @@ Open Y
 Paragraph
 : A Drupal data structure that allows content editors to create sections of content on a page using preset groups of fields. See [Paragraphs on Drupal.org](https://www.drupal.org/project/paragraphs) and [Paragraphs in the distribution](/docs/user-documentation/paragraphs/).
 
+PEF
+: The "Program Event Framework", also known as PEF, consists of a group of content types and front-end displays that can be used to create related programs, activities, classes, and sessions.
```
Comment (froboy) [inline content/en/docs/glossary/_index.md:79]:
```suggestion
: The "[Program Event Framework](/docs/development/program-event-framework/)", also known as PEF, consists of a group of [content types](/docs/user-documentation/content-types/activity-class-session/) and [front-end displays](/docs/user-documentation/schedules/) that can be used to create related programs, activities, classes, and sessions.
```

**Review by froboy — CHANGES_REQUESTED** [review]
Thanks @andrebonon, great addition. Just one tweak here to add lots of juicy links.

**Comment by andrebonon** [comment]
Thanks for the suggestions. Committed.

File: content/en/docs/glossary/_index.md [hunk 2]
```diff
@@ -75,6 +75,9 @@ Open Y
 Paragraph
 : A Drupal data structure that allows content editors to create sections of content on a page using preset groups of fields. See [Paragraphs on Drupal.org](https://www.drupal.org/project/paragraphs) and [Paragraphs in the distribution](/docs/user-documentation/paragraphs/).
 
+PEF
+: The "[Program Event Framework](/docs/development/program-event-framework/)", also known as PEF, consists of a group of [content types](/docs/user-documentation/content-types/activity-class-session/) and [front-end displays](/docs/user-documentation/schedules/) that can be used to create related programs, activities, classes, and sessions.
```
Comment (froboy) [inline content/en/docs/glossary/_index.md:79]: SCR-20240705-mlac

In the [preview](https://deploy-preview-89--sparkling-souffle-4f78b0.netlify.app/docs/glossary/) something about the link processor is adding an extra space before the link, so just removing those quotes. 

```suggestion
: The [Program Event Framework](/docs/development/program-event-framework/), also known as PEF, consists of a group of [content types](/docs/user-documentation/content-types/activity-class-session/) and [front-end displays](/docs/user-documentation/schedules/) that can be used to create related programs, activities, classes, and sessions.
```

**Comment by froboy** [comment]
Thanks @andrebonon. One tiny change there based on the preview, but otherwise good to go!

**Pull request merged**